src/DegreeOfImportanceComponent.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './DegreeOfImportanceComponent.css';
+
+class DegreeOfImportanceComponent extends React.Component{
+  constructor(props){
+    super(props);
+  }
+  render(){
+    return(
+      <div id='DegreeOfImportance'>
+        <h3>Фильтры</h3>
+        <ul>
+          <li onClick={()=>{this.props.sortData('1', this.props.items)}}>Степень важности 1</li>
```

Comment:
Extract priorities into array and pass them from parent into DegreeOfImportanceComponent and ImportanceComponent in order to avoid code duplication